x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_table.tsx

```diff
@@ -112,6 +118,22 @@ const columns: Array<EuiTableFieldDataColumnType<CspFindingsByResource>> = [
     ),
     render: (sections: string[]) => sections.join(', '),
   },
+  {
+    field: 'cluster_id',
+    name: (
+      <ColumnNameWithTooltip
+        columnName={TEXT.CLUSTER_ID}
+        tooltipContent={i18n.translate(
+          'xpack.csp.findings.resourceTable.resourceTableColumn.clusterIdColumnTooltipLabel',
```

Comment:
nit: suggesting to refactor tooltip into `TEXT` to be re-used like the column names